src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -44,6 +49,18 @@ public class HttpUtil {
 
   private static CloseableHttpClient httpClient;
 
+  private static final int DEFAULT_CONNECTION_TIMEOUT_MINUTES = 1;
+  private static final int DEFAULT_HTTP_CLIENT_SOCKET_TIMEOUT_MINUTES = 5;
+
+  private static final int DEFAULT_MAX_CONNECTIONS_PER_ROUTE = 100;
+  private static final int DEFAULT_MAX_CONNECTIONS = 100;
```

Comment:
may be add a comment about why 100 is chosen for `DEFAULT_MAX_CONNECTIONS` (because we only have 1 route for ingest)

Reply:
Will do!